conf/config-default-dev.yaml

```diff
@@ -0,0 +1,203 @@
+#
```

Comment:
should this file be added manually? cc @nic-chen

Reply:
How do I resolve the CI error?

Reply:
I do not think we need `conf/config-default-dev.yaml` for CI.
https://github.com/apache/apisix/pull/2339/checks?check_run_id=1223679517#step:6:310

it seems that you only need to fix the test case `t/admin/plugins.t` .

```
Test Summary Report
-------------------
t/admin/plugins.t                       (Wstat: 256 Tests: 15 Failed: 1)
  Failed test:  2
  Non-zero exit status: 1